laalaa/apps/advisers/tasks.py

```diff
@@ -229,15 +227,15 @@ def load_csv_into_db(self, csv_filename, headers, types):
         cursor.execute("DELETE FROM {table}".format(table=table_name))
 
         psql_command = [
-            'export PGPASSWORD=%s &&' % settings.DATABASES['default']['PASSWORD'],
-            'psql',
-            settings.DATABASES['default']['NAME'],
-            '-U',
-            settings.DATABASES['default']['USER'],
-            '-h',
-            settings.DATABASES['default']['HOST'],
-            '-c',
-            '"\copy {table} FROM {filename} DELIMITER \',\' CSV HEADER;"'.format(
+            "export PGPASSWORD=%s &&" % settings.DATABASES["default"]["PASSWORD"],
+            "psql",
+            settings.DATABASES["default"]["NAME"],
+            "-U",
+            settings.DATABASES["default"]["USER"],
+            "-h",
+            settings.DATABASES["default"]["HOST"],
+            "-c",
+            "\"\\copy {table} FROM {filename} DELIMITER ',' CSV HEADER;\"".format(
```

Comment:
💭 There were issues with changes to this command during the linter changes. Might be worth to double check with template-deploy too.

Reply:
I've tested this staging and working as expected